src/scripts/underground/Underground.ts

```diff
@@ -151,6 +151,14 @@ class Underground implements Feature {
         }
     }
 
+    public static getMineItemByName(name: string): UndergroundItem {
+        for (const item of UndergroundItem.list) {
+            if (item.name == name) {
+                return item;
+            }
+        }
```

Comment:
Could we simplify this with a find?
```suggestion
        return UndergroundItem.list.find(i => i.name == name);
```